qiskit_experiments/framework/backend_data.py

```diff
@@ -15,6 +15,7 @@
 Since `BackendV1` and `BackendV2` do not share the same interface, this
 class unifies data access for various data fields.
 """
+# pylint: disable=no-name-in-module
 from qiskit.providers.models import PulseBackendConfiguration
```

Comment:
Better to disable only this one line. I wouldn't bother suggesting if everything else was good but I want to take out the print statements I added as well.

```suggestion
from qiskit.providers.models import PulseBackendConfiguration  # pylint: disable=no-name-in-module
```